apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -343,6 +343,11 @@ test_exit_code=0
 readonly testlog=$test_tmp_dir/test.log
 
 test_file=$(file "$test_tmp_dir/$test_bundle_name.xctest/$test_bundle_name")
+if [[ "$test_type" = "XCUITEST" ]]; then
+  plugins_path="$test_tmp_dir/$runner_app/PlugIns"
+  test_file=$(file "$plugins_path/$test_bundle_name.xctest/$test_bundle_name")
+fi
```

Comment:
Is this necessary given the `plugins_path="$test_tmp_dir/$runner_app/PlugIns"` above?

If not:
```suggestion
test_file=$(file "$test_tmp_dir/$test_bundle_name.xctest/$test_bundle_name")
if [[ "$test_type" = "XCUITEST" ]]; then
  test_file=$(file "$plugins_path/$test_bundle_name.xctest/$test_bundle_name")
fi
```

If it's necessary or maybe just do it this way, it might be nice to set a var like this above:

```diff
 test_bundle_path="%(test_bundle_path)s"
 test_bundle_name=$(basename_without_extension "$test_bundle_path")
+test_bundle_binary="$test_tmp_dir/$test_bundle_name.xctest/$test_bundle_name"
```
And then update it when we `mv` it into that `PlugIns` dir above:

```diff
    mkdir -p "$plugins_path"
    mv "$test_tmp_dir/$test_bundle_name.xctest" "$plugins_path"
+   test_bundle_binary="$plugins_path/$test_bundle_name.xctest/$test_bundle_name"
```
Then:
```suggestion
test_file=$(file "$test_bundle_binary")
```

Reply:
Yes, thanks for the suggestions. I updated to the second one here which results in the following successful output:

```bash
++ file /tmp/bazel_tmp/test_tmp_dir/AddUserPluginTests.xctest/AddUserPluginTests
+ test_file='/tmp/bazel_tmp/test_tmp_dir/AddUserPluginTests.xctest/AddUserPluginTests: Mach-O 64-bit bundle arm64'
+ intel_simulator_hack=false
+ architecture=arm64
++ arch
+ [[ arm64 == arm64 ]]
+ [[ /tmp/bazel_tmp/test_tmp_dir/AddUserPluginTests.xctest/AddUserPluginTests: Mach-O 64-bit bundle arm64 != *arm64* ]]
+ should_use_xcodebuild=false
```